lib/ttdrest/concerns/campaign.rb

```diff
@@ -50,6 +50,11 @@ def build_campaign_data(campaign_id, name, budget, start_date, campaign_conversi
           campaign_data.delete("CampaignConversionReportingColumns") if campaign_conversion_reporting_columns.empty?
         end
 
+        # On create only, support passing in additional_fee_card
+        if campaign_id.nil? && !params[:additional_fee_card].nil?
```

Comment:
Do we want to accept empty values for this? If not and we've got activesupport, then `.present?` would fit the bill

Reply:
👍